### PR TITLE
Update the Type inference section's link

### PR DESCRIPTION
**Pull request by chagweyh**

### CODE REVIEW

**Review by Philippus — APPROVED** [review]
LGTM

File: _overviews/scala3-book/scala-features.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ In that list:
 Scala’s type system enforces, at compile-time, that abstractions are used in a safe and coherent manner.
 In particular, the type system supports:
 
-- [Type inference](/tour/type-inference.html)
+- [Type inference]({% link _overviews/scala3-book/types-inferred.md %})
```
Comment (Philippus) [inline _overviews/scala3-book/scala-features.md:164]:
```suggestion
- [Inferred types]({% link _overviews/scala3-book/types-inferred.md %})
```

**Comment by Philippus** [comment]
@alvinj do you agree with this change?

**Comment by alvinj** [comment]
> @alvinj do you agree with this change?

Sorry for the delay, but yes, that looks good to me! 👍

**Pull request merged**